corehq/apps/products/fixtures.py

```diff
@@ -18,16 +18,16 @@
 
 
 def product_fixture_generator_json(domain):
-    if SQLProduct.objects.filter(domain=domain).count() == 0:
+    if not SQLProduct.objects.filter(domain=domain).exists():
         return None
 
     fields = filter(lambda x: x != CUSTOM_DATA_SLUG, PRODUCT_FIELDS)
     fields.append('@id')
 
-    custom_fields = CustomDataFieldsDefinition.get_by_domain_and_type(domain, 'ProductFields')
+    custom_fields = get_by_domain_and_type(domain, 'ProductFields')
     if custom_fields:
-        for f in custom_fields.fields:
-            fields.append(CUSTOM_DATA_SLUG + '/' + f.slug)
+        for f in custom_fields['doc']['fields']:
```

Comment:
What's the advantage of passing around the unwrapped doc?  It seems like it'd just be simpler to let jsonobject worry about it (although I don't think this data will be messy, it's schema'd and hasn't changed that I'm aware of).